prov/tcp/src/tcpx_ep.c

```diff
@@ -747,3 +808,24 @@ int tcpx_passive_ep(struct fid_fabric *fabric, struct fi_info *info,
 	free(_pep);
 	return ret;
 }
+
+int tcpx_set_port_range ()
```

Comment:
Why is this called after selecting the port? Can we combine this function with tcpx_init_env() and call from init?